### ISSUE TITLE
Weird black tail on renamed script variables

### ISSUE THREAD

**Issue by brianharvey** [jmoenig/Snap#2463]
![U4L5-Compression script pic](https://user-images.githubusercontent.com/3885074/61603159-67e46480-abf1-11e9-89af-3e9cec4533f3.png)

Note that it only happened on some of them.  Dunno why.

**Comment by jmoenig** [comment]
that's refreshing artifact for sure. But, honestly, the renaming variables features is pretty broken and probably needs to be rewritten completely...

**Comment by brianharvey** [comment]
It survived closing and reopening the Block Editor.

**Comment by jmoenig** [comment]
Yes, because the changed and refreshed block in the definition body is just ... a block, and doesn't get re-created when you open the block editor again - it just shows you the same old block you've seen before :-)
